MobSF/utils.py

```diff
@@ -503,8 +503,9 @@ def check_basic_env():
 
 def first_run(secret_file, base_dir, mobsf_home):
     # Based on https://gist.github.com/ndarville/3452907#file-secret-key-gen-py
-
-    if is_file_exists(secret_file):
+    if 'MOBSF_SECRET_KEY' in os.environ:
```

Comment:
Why do you need a static secret? If you just need a predefined REST `API_KEY` , use this env variable instead.

https://github.com/MobSF/Mobile-Security-Framework-MobSF/blob/f56135bcc751c451142de5d8b8947690ea5d5b46/MobSF/utils.py#L72

Reply:
Sorry, I don't really understand. What do you mean by 'static secret'? It is static right now because secret file is generated when the image is built, not on container startup. As a result, API_KEY is always the same by default as well. What I suggest is an option to set secret key on container run using environment variable

Reply:
I got confused, yes you are right.